assets/caniuse.css

```diff
@@ -85,3 +93,8 @@ see https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md for stats */
 .caniuse-stats .caniuse-browser:hover > ul {
   display: block;
 }
+
+.display_emoji {
```

Comment:
I think it should be possible to just use `@media print` rule without introducing a new emoji rule. As mentioned below, use "CSS Generated Content" to dynamically add the emojis without needing to touch the JS.